pkg/kubelet/dockertools/runner.go

```diff
@@ -0,0 +1,133 @@
+/*
+Copyright 2014 Google Inc. All rights reserved.
```

Comment:
Yah, no idea how that got in
On Fri, Mar 27, 2015 at 6:17 PM Yifan Gu <notifications@github.com> wrote:

> In pkg/kubelet/dockertools/runner.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/6100#discussion_r27335103>
> :
>
> > @@ -0,0 +1,133 @@
> > +/*
> > +Copyright 2014 Google Inc. All rights reserved.
>
> 2015?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/6100/files#r27335103>
> .
>